src/app/server/Server.cpp

```diff
@@ -138,7 +138,7 @@ CHIP_ERROR Server::Init(AppDelegate * delegate, uint16_t secureServicePort, uint
     app::DnssdServer::Instance().SetCommissioningModeProvider(&mCommissioningWindowManager);
 
     // Group data provider must be initialized after mDeviceStorage
-    err = mGroupsProvider.Init();
+    err = mGroupsProvider.Init(&mDeviceStorage);
```

Comment:
I don't see how this fixes https://github.com/project-chip/connectedhomeip/issues/16028, which is not even about the groups data provider as far as I can tell.  What am I missing?

Reply:
It doesn't fix 16028. I think it's a mistake the fixes line in the description. The clean-up of Server dependencies to storage is on the way to single storage injection points for Server and controller, so it relates to 16028 but doesn't fix it

Reply:
You're both correct. I have replace the word "Fix" with "Relates" in the description.